tmt.spec

```diff
@@ -198,6 +217,13 @@ chmod 1777 %{buildroot}%{workdir_root}
 %files provision-virtual
 %{python3_sitelib}/%{name}/steps/provision/{,__pycache__/}testcloud.*
 
+%if 0%{?rhel} >= 9 || 0%{?fedora}
+
+%files provision-beaker-mrack
+%{python3_sitelib}/%{name}/steps/provision/{,__pycache__/}beaker_mrack.*
+%config(noreplace) %{_sysconfdir}/%{name}/mrack*
+%endif
```

Comment:
It's also necessary to exclude `mrack` files from the `python3-tmt` package:

```
rpm -qlp ./tmp/RPMS/noarch/python3-tmt-1.20.0-1.fc36.noarch.rpm  | grep mrack
/usr/lib/python3.10/site-packages/tmt/steps/provision/__pycache__/beaker_mrack.cpython-310.opt-1.pyc
/usr/lib/python3.10/site-packages/tmt/steps/provision/__pycache__/beaker_mrack.cpython-310.pyc
/usr/lib/python3.10/site-packages/tmt/steps/provision/beaker_mrack.py
/usr/lib/python3.10/site-packages/tmt/steps/provision/mrack
/usr/lib/python3.10/site-packages/tmt/steps/provision/mrack/mrack-provisioning-config.yaml
/usr/lib/python3.10/site-packages/tmt/steps/provision/mrack/mrack.conf
```

Reply:
Will do